care/emr/resources/report/report_upload/spec.py

```
@@ -0,0 +1,128 @@
+class ReportUploadUpdateSpec(ReportUploadBaseSpec):
+    pass
+
+
+class ReportUploadCreateSpec(ReportUploadBaseSpec):
+    template: str
+    original_name: str
+    report_type: str
+    associating_id: str
+    mime_type: str
+
+    def perform_extra_deserialization(self, is_update, obj):
```

Ruff (0.14.4)

Unused method argument: `is_update`

(ARG002)